src/crawlee/errors.py

```diff
@@ -89,13 +88,3 @@ def __init__(self, wrapped_exception: Exception, crawling_context: BasicCrawling
 @docs_group('Errors')
 class ContextPipelineInterruptedError(Exception):
     """May be thrown in the initialization phase of a middleware to signal that the request should not be processed."""
-
-
-@docs_group('Errors')
-class ServiceConflictError(RuntimeError):
```

Comment:
Do you have any other plans to prevent services from being overwritten from multiple call sites? I think it might be useful to keep it.

Reply:
`ServiceConflictError` is back